WikiWatershed/model-my-watershed
# Add support for BiG CZ domains

## Overview

Replace the secondary domain support of azavea.com subdomains with portal.bigcz.org domains. Most of the hard work was already done in adding support for secondary domains. This just tweaks Django settings so that the new domains are whitelisted.

Closes https://github.com/WikiWatershed/model-my-watershed/issues/1908

## Testing Instructions

Navigate to the staging site via https://staging.portal.bigcz.org/ and click through the certificate warning (this will remain until certificates for the `portal.bigcz.org` domains go through). You should be able to see the Model My Watershed website.
